docs/OncoKB-Data-Access.md

```diff
@@ -0,0 +1,13 @@
+# Introduction
+
+A data access token is required to use OncoKB. Please review OncoKB [terms](https://www.oncokb.org/terms) 
```

Comment:
maybe good to specify from what cBioPortal version this will be a requirement. We could rephrase it to "a data access token is required **soon**. Then we can merge this to master. The master branch gets automatically deployed to docs.cbioportal.org